lib/rules/web/admin_console/4.3/deployment.xyaml

```diff
@@ -24,10 +24,10 @@ confirm_cancel_rollout:
   action: click_button
 search_image:
   params:
-    clear_field_id: form-input-search-searchTerm-field
+    field_css: input#form-input-search-searchTerm-field
     input_field_id: form-input-search-searchTerm-field
     input_value: <search_content>
-  action: clear_input_value
+  action: clear_value
```

Comment:
[Failed log](http://ci-qe-openshift.usersys.redhat.com/userContent/cucushift/v3/2019/12/24/22:19:19/Check_basic_process_of_create_delete_DC_from_image/console.html)   Because of `op: clear` operation in the `clear_input_value` did not clear value at all, then the search button still be in disable status.  I met many times the `op: clear`  did not take the effect but not find the root cause. So add the new rules as kind of workaround for clear something